src/mudlet.cpp

```diff
@@ -714,246 +418,611 @@ mudlet::mudlet()
+                currentTranslation.mNativeName = QStringLiteral("Español");
+            } else if (!languageCode.compare(QLatin1String("pt_PT"), Qt::CaseInsensitive)) {
+                currentTranslation.mNativeName = QStringLiteral("Portugês");
+            } else {
+                currentTranslation.mNativeName = languageCode;
+            }
+            currentTranslation.mMudletTranslationFileName = translationFileName;
+            mTranslationsMap.insert(languageCode, currentTranslation);
+        } else {
+            // This is very unlikely to be reached as it means that a file that
+            // matched the naming to be a Mudlet translation file was not infact
+            // one...
+            qDebug().noquote().nospace() << "    no Mudlet translation found for locale code: \"" << languageCode << "\".";
+        }
+        // As pQtTranslator goes out of scope at the end of each loop it will
+        // dispose of the temporarily created QTranslator for us...
+    }
+}
+
+// To be used AFTER scanForMudletTranslations(...) has been called, this will
+// insert the corresponding Qt system translation pathFileNames (path to
+// filenames: qt_xx.qm or qt_xx_YY.qm) into the entries in the mTranslationsMap
+// QMap.
+void mudlet::scanForQtTranslations(const QString& path)
+{
+    mQtTranslationsPathName = path;
+    qDebug().nospace().noquote() << "mudlet::scanForQtTranslations(\"" << path << "\") INFO - Seeking Qt translation files:";
+    QMutableMapIterator<QString, translation> itTranslation(mTranslationsMap);
+    while (itTranslation.hasNext()) {
+        itTranslation.next();
+        const QString languageCode = itTranslation.key();
+        auto* pTranslator = new QTranslator();
+        std::unique_ptr<QTranslator> pQtTranslator(pTranslator);
+        QString translationFileName(QStringLiteral("qt_%1.qm").arg(languageCode));
+        if (pQtTranslator->load(translationFileName, path)) {
+            qDebug().noquote().nospace() << "    found a Qt translation for locale code: \"" << languageCode << "\"";
+            /*
+             * Unfortunately, success in this operation does not mean that
+             * a qt_xx_YY.qm translation file has been located, as the
+             * (bool) QTranslator::load(...)
+             * call can forget about both the _YY and even the _xx if filenames
+             * with those elements are not found but a less detailed filename
+             * IS detected.
+             *
+             * So although we can note the load of a given pathFileName is
+             * sucessful it might not be exactly what it seems to be!
+             */
+            translation current = itTranslation.value();
+            current.mQtTranslationFileName = translationFileName;
+            itTranslation.setValue(current);
+        } else {
+            qDebug().noquote().nospace() << "    no Qt translation found for locale code: \"" << languageCode << "\"";
```

Comment:
Yes, for instance on my Linux system it is telling me that I do not have any Qt translations suitable for the `el_GR` (Greek), `nl_NL` (Netherlands) and `zh_TW` Taiwanese locales... it may be particularly helpful currently for the Chinese case given that it is not clear that Qt/AppVeyor provides a `zh_CN` Qt translation for Windows, even though Linux distributions do. :confused:

Reply:
It's quite spammy and doubles what Mudlet prints on startup, so once you get it working it'd be nice to remove it.